tracim/tracim/tests/library/test_thread.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+import transaction
+from nose.tools import eq_
+
+from tracim.tests import BaseTestThread, TestStandard
+
+
+class TestThread(BaseTestThread, TestStandard):
+
+    def test_create_thread(self, key='1'):
+        return self._create_thread(
```

Comment:
In really tests are made in ``self._create_thread``. Idea here is to have a test of this procedure. If all "thread" tests fails and this test fail, we have to look in thread creation before other debug. (And yes, more test about thread should be wrote ^^)